recipes/r-getip/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set version = '0.1-3' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-getip
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/getip_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/getip/getip_{{ version }}.tar.gz
+  sha256: 0ec1677309dc3f2d861d61e7b3de0e74d1fc94aefd03bc5a6c27059e30f3cc24
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
```

Fix DSO issue (following [proposed fix](https://github.com/conda/conda-build/pull/4786)):

```suggestion
  missing_dso_whitelist:
    - '*/R.dll'           # [win]
  number: 0
```